easysnmp/interface.c

```diff
@@ -1142,12 +1142,12 @@ static int __send_sync_pdu(netsnmp_session *ss, netsnmp_pdu *pdu,
                            bitarray *invalid_oids)
 {
     int status = 0;
-    long command = pdu->command;
+    int command = pdu->command;
     char *tmp_err_str;
     size_t retry_num = 0;
 
     /* Note: SNMP uses 1-based indexing with OIDs, so 0 is unused */
-    unsigned long last_errindex = 0;
```

Comment:
This is unnecessary churn. Could you please change the cast back to `unsigned long` here?

Reply:
reverted